docs/modeling/how-to-extend-the-domain-specific-language-designer.md

```diff
@@ -82,8 +82,6 @@ To uninstall the extension, follow these steps:
 
 To make a shortcut menu command appear on the DSL Designer surface or in the DSL Explorer window, write a class resembling the following.
 
-The class must implement `ICommandExtension` and must have the attribute `DslDefinitionModelCommandExtension`.
```

Comment:
Why were these three lines deleted in this file?

Reply:
My guess would be from a rebase rather than intentional

Reply:
Looks like the rebase got confused with https://github.com/MicrosoftDocs/visualstudio-docs/commit/53fa78032c0272147929db3d0d10cb11a80543a8
I did this a few weeks ago and forgot, then rebased yesterday